services/minespace-web/src/components/pages/Project/ProjectDescriptionTab.tsx

```diff
@@ -0,0 +1,279 @@
+import React, { useEffect, useState } from "react";
+import { Row, Col, Typography, Button, Alert, Badge } from "antd";
+import Callout from "@/components/common/Callout";
+import { CALLOUT_SEVERITY } from "@mds/common/constants/strings";
+import CoreTable from "@mds/common/components/common/CoreTable";
+import { getProject } from "@mds/common/redux/selectors/projectSelectors";
+import { useSelector } from "react-redux";
+
+import { getPermits } from "@mds/common/redux/selectors/permitSelectors";
+import {
+  renderTextColumn,
+  renderDateColumn,
+} from "@mds/common/components/common/CoreTableCommonColumns";
+import { IAuthorizationSummaryColumn } from "@mds/common/interfaces";
+import { Link } from "react-router-dom";
+
+import {
+  getDropdownProjectSummaryPermitTypes,
+  getTransformedProjectSummaryAuthorizationTypes,
+} from "@mds/common/redux/selectors/staticContentSelectors";
+import { ColumnsType } from "antd/es/table";
+import * as routes from "@/constants/routes";
+import { useHistory } from "react-router-dom";
+import { formatTime, formatDateTimeTz } from "@mds/common/redux/utils/helpers";
+
+export const ProjectDescriptionTab = () => {
+  const history = useHistory();
+  const [minesActData, setMinesActData] = useState([]);
+  const [environmentalManagementActData, setEnvironmentalManagementActData] = useState([]);
+  const [waterSustainabilityActData, setWaterSustainabilityActData] = useState([]);
+  const [forestryActData, setForestryActData] = useState([]);
+
+  const permits = useSelector(getPermits);
+  const project = useSelector(getProject);
+  const transformedProjectSummaryAuthorizationTypes = useSelector(
+    getTransformedProjectSummaryAuthorizationTypes
+  );
+  const dropdownProjectSummaryPermitTypes = useSelector(getDropdownProjectSummaryPermitTypes);
+  const notApplicableText = "N/A";
+
+  let processedOtherActPermitResult: any[] = [];
+
+  const createStatusColumn = (text: string) => ({
+    title: "Status",
```

Comment:
column should have a key attribute. If it's going to be for submitted or failed, probably good to use that for the key.